app/views/layouts/dashboard/_sidebar.html.erb

```diff
@@ -78,7 +78,7 @@
         context: {organization: Current.organization})
       %>
         <li class="nav-item">
-          <%= active_link_to staff_custom_form_forms_path, class: 'nav-link' do %>
+          <%= active_link_to '#', class_active: 'disabled', class: 'nav-link disabled' do %>
```

Comment:
I believe this should be deleted (not just disabled). Otherwise this PR is good to go. Thanks!

Reply:
Sorry, I should have been more specific. Remove the whole option from the menu please.

Reply:
sure